addons/event/models/event_mail.py

```diff
@@ -91,7 +91,10 @@ def _compute_done(self):
             if mail.interval_type in ['before_event', 'after_event']:
                 mail.done = mail.mail_sent
             else:
-                mail.done = len(mail.mail_registration_ids) == len(mail.event_id.registration_ids) and all(mail.mail_sent for mail in mail.mail_registration_ids)
+                mail.done = len(mail.mail_registration_ids) and (
```

Comment:
What is the purpose of that change ? It seem the only change would be with a void mail.mail_registration_ids (from True to False), is it intended ?

Reply:
Yes but thinking back I am not sure this is a good idea. the "done" field was true when there was no registration (so no mail to send) and is used in the view in a column that is called "Sent" which is a checkbox and represents the fact that all mails are sent. So, do you think that this field should stay true in case there was no mail to send ?

This is a screenshot when this change is applied. Before, the checkbox was checked. This is a new event and I am in edit mode, but it may implies that "mails have already been sent"
![event_edition](https://user-images.githubusercontent.com/17027356/106739217-2b345e80-6619-11eb-9db5-1787faae4555.png)